com.authy.Authy.metainfo.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+    <id>com.authy.Authy</id>
+    <metadata_license>CC-BY-SA</metadata_license>
```

Comment:
this is an invalid metadata license,

also we really encourage to use `CC0-1.0`

Reply:
Thanks for the report, it was an initial example copied from NordPass.
I will fix it in next commit, still want to do some more tests.